Tests/ApolloCodegenTests/ApolloSchemaPublicTests.swift

```diff
@@ -1,9 +1,9 @@
 import XCTest
 import ApolloTestSupport
 import ApolloCodegenTestSupport
-@testable import ApolloCodegenLib
+import ApolloCodegenLib
```

Comment:
Changed how this test imports `ApolloCodegenLib` to ensure `public` access is caught in further tests.